agent/config/config.go

```diff
@@ -510,6 +518,17 @@ const (
 	defaultPolicyEvalAckTimeout = 5 * time.Minute
 )
 
+// TODO: there's an unexpected import cycle that prevents us from using the
+// values defined in policy/source.go.
```

Comment:
A candidate to move to the `sdk` package?

Reply:
I'm not sure to be honest (hence the `TODO` to punt it to future Luiz 😬).

I think the root of the problem is this import: https://github.com/hashicorp/nomad-autoscaler/blob/v0.3.3/sdk/helper/nomad/config.go#L7

The `sdk` package should never require anything outside itself (and I thought we had a check for that), so whole agent config is probably what we would want to move to `sdk`, but that would require quite a bit of refactoring (mostly copy and paste though).